docs/index.md

```diff
@@ -8,21 +8,21 @@ date: 2024-10-22
 ---
 # Welcome
 
-Welcome to the Department for Education Secure-by-Design development documentation.
+Welcome to the Department for Education (DfE)'s Secure by Design (SbD) site for development documentation.
```

Comment:
I put development here as in "sbd development" rather than "for development", but thinking about it, maybe we should just end it as:

> Welcome to the Department for Education (DfE)'s Secure by Design (SbD) documentation site.